app/src/main/java/org/falaeapp/falae/fragment/SpreadSheetFragment.kt

```diff
@@ -58,21 +58,14 @@ class SpreadSheetFragment : Fragment() {
     override fun onOptionsItemSelected(item: MenuItem): Boolean {
         when (item.itemId) {
             R.id.remove_item -> {
-                val confirmBuilder = AlertDialog.Builder(activity)
-                val confirmDialog = confirmBuilder.setTitle(getString(R.string.removeUser))
-                        .setMessage(getString(R.string.questionRemoveUser))
-                        .setPositiveButton(getString(R.string.yes_option), { _, _ ->
-                            mListener.removeUser(user!!)
-                        })
-                        .setNegativeButton(getString(R.string.no_option), { _, _ -> })
-                        .create()
-                confirmDialog.setOnShowListener {
-                    val buttonPositive = confirmDialog.getButton(AlertDialog.BUTTON_POSITIVE)
-                    val buttonNegative = confirmDialog.getButton(AlertDialog.BUTTON_NEGATIVE)
-                    buttonPositive.setTextColor(resources.getColor(R.color.colorAccent))
-                    buttonNegative.setTextColor(resources.getColor(R.color.colorAccent))
-                }
-                confirmDialog.show()
+                Util.createDialog(
+                        context = context,
```

Comment:
why do you need it here?

Reply:
It's for user deletion.
I moved this peace of code to the Util class.